### ISSUE TITLE
✅ 🐞 Custom Taxonomy display is now shown correctly when Element Caching is Active

### ISSUE THREAD

**Issue by musikblog** [elementor/elementor#29626]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [ ] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

3.26.0 breaks our custom taxonomies, e.g. the artist tag, which is displayed above the post image (so you can verify yourself!). Instead of the correct artist, Elementor now shows a random artist, the same for EVERY post!

![Screenshot 2024-12-18 at 09 08 21](https://github.com/user-attachments/assets/562fdd26-c77c-4f75-836c-9dffff2c6fc8)
![Screenshot 2024-12-17 at 10 19 59](https://github.com/user-attachments/assets/579c311d-0ccb-4aef-91ce-eeeba70e93b8)

### Steps to reproduce

1. Create custom taxonomy, e.g. artist
2. Create 3 artists, e.g. Taylor Swift, Nick Cave, The National
3. Create 3 posts and with a different artist assigned to each post
4. View these 3 posts with V3.25.11 and V3.26.0

### Expected behavior

Taxonomies are correctly displayed

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt


== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Percona Server (GPL), Release '28', Revision '47601f19'$ v8.0.36-28
	PHP Version: 8.2.26
	PHP Memory Limit: 1024M
	PHP Max Input Vars: 20000
	PHP Max Post Size: 2048M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.6.2
	Site URL: https://www.musikblog.de
	Home URL: https://www.musikblog.de
	WP Multisite: No
	Max Upload Size: 2 GB
	Memory limit: 1024M
	Max Memory limit: 1024M
	Permalink Structure: /%year%/%monthnum%/%postname%/
	Language: de_DE
	Timezone: Europe/Berlin
	Debug Mode: Inactive

== Theme ==
	Name: MusikBlog Elementor
	Version: 2.3.1.1615025338
	Author: MusikBlog
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 3.2.1
	Parent Theme Author: Elementor Team

== User ==
	Role: 
	WP Profile lang: de
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:133.0) Gecko/20100101 Firefox/133.0

== Active Plugins ==
	Admin and Site Enhancements (ASE)
		Version: 7.6.2
		Author: wpase.com

	Admin Columns Pro
		Version: 6.4.13
		Author: AdminColumns.com

	Advanced Custom Fields
		Version: 6.3.11
		Author: WP Engine

	Akismet Anti-spam: Spam Protection
		Version: 5.3.5
		Author: Automattic - Anti-spam Team

	AMP
		Version: 2.5.5
		Author: AMP Project Contributors

	Code Snippets Pro (Premium)
		Version: 3.6.6.1
		Author: Code Snippets Pro

	CookieYes | GDPR Cookie Consent
		Version: 3.2.8
		Author: CookieYes

	Database Manager - WP Adminer
		Version: 3.0.3.1
		Author: Pexle Chris

	Elementor
		Version: 3.25.11
		Author: Elementor.com

	Elementor Pro
		Version: 3.25.5
		Author: Elementor.com

	FS Poster
		Version: 6.6.5
		Author: FS-Code

	GA Google Analytics
		Version: 20241102
		Author: Jeff Starr

	Google for WooCommerce
		Version: 2.9.2
		Author: WooCommerce

	Gravity Forms
		Version: 2.9.1
		Author: Gravity Forms

	Gravity Forms + Custom Post Types
		Version: 3.1.29
		Author: Gravity Wiz

	Gravity Forms Akismet Add-On
		Version: 1.0
		Author: Gravity Forms

	Gravity Forms Survey Add-On
		Version: 4.1.0
		Author: Gravity Forms

	Jetpack
		Version: 14.1
		Author: Automattic

	Jetpack Boost
		Version: 3.6.1
		Author: Automattic - Jetpack Site Speed team

	Loco Translate
		Version: 2.6.14
		Author: Tim Whitlock

	MailPoet
		Version: 5.5.1
		Author: MailPoet

	MailPoet Premium
		Version: 5.5.0
		Author: MailPoet

	Minimum Periods for WooCommerce Subscriptions
		Version: 2.7
		Author: WPExtend

	PDF Invoices & Packing Slips for WooCommerce
		Version: 3.9.2
		Author: WP Overnight

	Post SMTP
		Version: 2.9.14
		Author: Post SMTP

	Push Notifications for WP - Self Hosted Web Push Notifications
		Version: 1.39
		Author: Magazine3

	Sanitize Media Filenames
		Version: 1.0
		Author: Rafael Chaves Freitas

	SearchWP Live Ajax Search
		Version: 1.8.3
		Author: SearchWP, LLC

	SoundCloud Shortcode
		Version: 4.0.3
		Author: SoundCloud Inc., Lawrie Malen

	Super Progressive Web Apps
		Version: 2.2.32
		Author: SuperPWA

	UpdraftPlus - Backup/Restore
		Version: 2.24.11.1
		Author: TeamUpdraft, DavidAnderson

	User Switching
		Version: 1.9.1
		Author: John Blackbourn

	WooCommerce
		Version: 9.5.1
		Author: Automattic

	WooCommerce.com Update Manager
		Version: 1.0.3
		Author: Automattic

	WooCommerce Memberships
		Version: 1.26.9
		Author: SkyVerge

	WooCommerce Order Status Control
		Version: 1.16.0
		Author: SkyVerge

	WooCommerce PayPal Payments
		Version: 2.9.5
		Author: WooCommerce

	WooCommerce Subscriptions
		Version: 7.0.0
		Author: WooCommerce

	WooPayments
		Version: 8.6.1
		Author: WooCommerce

	WordPress Editorial Calendar
		Version: 3.8.8
		Author: Editorial Calendar Team

	WP-Optimize - Clean, Compress, Cache
		Version: 3.8.0
		Author: TeamUpdraft, DavidAnderson

	WP Rollback
		Version: 2.0.7
		Author: WP Rollback

	XML Sitemap & Google News
		Version: 5.4.9
		Author: RavanH

	Yoast SEO
		Version: 24.0
		Author: Team Yoast

	[Musikblog] Favoriten und Jahres-Charts
		Version: 1.0
		Author: Jöran Kuschel


== Must-Use Plugins ==
	Elementor Safe Mode
		Version: 1.0.0
		Author: Elementor.com

	pexlechris_adminer_avoid_conflicts_with_other_plugins.php
		Version: 3.0.3
		Author: 

	Raidboxes Must-Use Plugins
		Version: 0.1
		Author: Raidboxes


== Eigenschaften ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	woocommerce: Aktiv


== Elementor-Experimente ==
	Improved CSS Loading: Standardmäßig deaktiviert
	Inline-Schriftarten-Symbole: Aktiv
	Additional Custom Breakpoints: Standardmäßig aktiviert
	Container: Aktiv
	Upgrade Swiper Library: Standardmäßig aktiviert
	Nested Elements Performance: Standardmäßig aktiviert
	Optimized Control Loading: Aktiv
	Optimized Markup: Standardmäßig deaktiviert
	Conditionally load Swiper CSS files: Standardmäßig deaktiviert
	Plugin Onboarding: Standardmäßig aktiviert
	CSS Smooth Scroll: Standardmäßig aktiviert
	Default to New Theme Builder: Standardmäßig aktiviert
	Header und Footer: Standardmäßig deaktiviert
	Elementor Home Screen: Standardmäßig aktiviert
	Startseiten: Inaktiv
	Nested Elements: Aktiv
	Editor Top Bar: Standardmäßig aktiviert
	Seiten Panel: Standardmäßig deaktiviert
	Element Caching: Standardmäßig deaktiviert
	Link In Bio: Standardmäßig aktiviert
	Floating Buttons: Standardmäßig aktiviert
	Elementor Editor Events: Standardmäßig deaktiviert
	Atomic Widgets: Standardmäßig deaktiviert
	Launchpad Checklist: Standardmäßig deaktiviert
	Menü: Standardmäßig deaktiviert


== Protokoll ==
	
Log: showing 20 of 282024-12-10 15:49:22 [info] Elementor/Upgrades - _on_each_version Finished 
2024-12-10 15:49:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.10',
  'to' => '3.25.11',
)]
2024-12-10 15:49:22 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.10',
  'to' => '3.25.11',
)]
2024-12-12 20:22:23 [info] elementor-pro::elementor_pro_updater Started 
2024-12-12 20:22:23 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-12-12 20:22:23 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-12-12 20:22:23 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.25.4',
  'to' => '3.25.5',
)]
2024-12-12 20:22:23 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.25.4',
  'to' => '3.25.5',
)]
2024-12-16 19:09:30 [info] elementor::elementor_updater Started 
2024-12-16 19:09:30 [info] Elementor/Upgrades - _on_each_version Start  
2024-12-16 19:09:30 [info] Elementor/Upgrades - _on_each_version Finished 
2024-12-16 19:09:30 [info] Elementor/Upgrades - _v_3_26_0 Start  
2024-12-16 19:09:30 [info] Elementor/Upgrades - _v_3_26_0 Finished 
2024-12-16 19:09:30 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.11',
  'to' => '3.26.0',
)]
2024-12-16 19:09:30 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.11',
  'to' => '3.26.0',
)]
2024-12-16 20:35:33 [info] elementor-pro::elementor_pro_updater Started 
2024-12-16 20:35:33 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-12-16 20:35:33 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-12-16 20:35:33 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.25.5',
  'to' => '3.26.0',
)]
2024-12-16 20:35:34 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.25.5',
  'to' => '3.26.0',
)]

PHP: showing 12 of 12PHP: 2024-11-21 10:01:53 [warning X 619][../wp-content/plugins/elementor/includes/conditions.php::87] Undefined array key "background_image" [array (
  'trace' => '
#0: ../wp-content/plugins/elementor/includes/conditions.php(87): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ../wp-content/plugins/elementor/includes/base/controls-stack.php(1438): Elementor\Conditions :: check()
#2: ../wp-content/plugins/elementor/includes/base/controls-stack.php(1222): Elementor\Controls_Stack -> is_control_visible()
#3: ../wp-content/plugins/elementor/includes/base/controls-stack.php(1227): Elementor\Controls_Stack -> get_active_settings()
#4: ../wp-content/plugins/elementor/includes/base/controls-stack.php(1263): Elementor\Controls_Stack -> get_active_settings()
',
)]
PHP: 2024-11-21 10:01:53 [warning X 619][../wp-content/plugins/elementor/includes/conditions.php::90] Trying to access array offset on value of type null [array (
  'trace' => '
#0: ../wp-content/plugins/elementor/includes/conditions.php(90): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ../wp-content/plugins/elementor/includes/base/controls-stack.php(1438): Elementor\Conditions :: check()
#2: ../wp-content/plugins/elementor/includes/base/controls-stack.php(1222): Elementor\Controls_Stack -> is_control_visible()
#3: ../wp-content/plugins/elementor/includes/base/controls-stack.php(1227): Elementor\Controls_Stack -> get_active_settings()
#4: ../wp-content/plugins/elementor/includes/base/controls-stack.php(1263): Elementor\Controls_Stack -> get_active_settings()
',
)]
PHP: 2024-11-21 20:59:32 [warning X 1][../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key "condition_type" [array (
  'trace' => '
#0: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: ../wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2024-11-21 20:59:33 [notice X 4][../wp-content/plugins/elementor-pro/modules/screenshots/module.php::33] Function utf8_decode() is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-11-21 20:59:58 [error X 1][../wp-content/plugins/elementor-pro/modules/screenshots/module.php::229] Uncaught WpOrg\Requests\Exception\Http\Status403: 403 Forbidden in ../wp-content/plugins/elementor-pro/modules/screenshots/module.php:229
Stack trace:
#0 ../wp-content/plugins/elementor-pro/modules/screenshots/module.php(264): ElementorPro\Modules\Screenshots\Module->is_screenshot_proxy_mode(Array)
#1 ../wp-content/plugins/elementor/core/base/module.php(85): ElementorPro\Modules\Screenshots\Module->__construct()
#2 ../wp-content/plugins/elementor-pro/core/modules-manager.php(98): Elementor\Core\Base\Module::instance()
#3 ../wp-content/plugins/elementor-pro/plugin.php(366): ElementorPro\Core\Modules_Manager->__construct()
#4 ../wp-includes/class-wp-hook.php(324): ElementorPro\Plugin->on_elementor_init('')
#5 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 ../wp-content/plugins/elementor/includes/plugin.php(685): do_action('...')
#8 ../wp-includes/class-wp-hook.php(324): Elementor\Plugin->init('')
#9 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#10 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#11 ../wp-settings.php(700): do_action('...')
#12 ../wp-config.php(69): require_once('...')
#13 ../wp-load.php(50): require_once('...')
#14 ../wp-blog-header.php(13): require_once('...')
#15 ../index.php(17): require('...')
#16 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-12-10 15:49:34 [warning X 283250][../wp-content/plugins/elementor/includes/conditions.php::90] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-12-16 19:09:30 [warning X 1][../wp-content/plugins/elementor/core/files/uploads-manager.php::647] rmdir(../wp-content/uploads/elementor/tmp/): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-12-16 19:09:30 [warning X 1][../wp-content/plugins/elementor/core/files/manager.php::112] unlink(../wp-content/uploads/elementor/css/custom-frontend.min.css): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-12-16 19:09:32 [warning X 1][../wp-content/plugins/elementor/core/files/manager.php::112] unlink(../wp-content/uploads/elementor/css/post-53813.css): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-12-17 10:38:10 [error X 1][../wp-content/plugins/elementor/core/documents-manager.php::121] Uncaught Error: Class "Elementor\Core\DocumentTypes\Post" not found in ../wp-content/plugins/elementor/core/documents-manager.php:121
Stack trace:
#0 ../wp-includes/class-wp-hook.php(324): Elementor\Core\Documents_Manager->register_default_types(Object(Elementor\Core\Documents_Manager))
#1 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#2 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#3 ../wp-content/plugins/elementor/core/documents-manager.php(728): do_action('...', Object(Elementor\Core\Documents_Manager))
#4 ../wp-content/plugins/elementor/core/documents-manager.php(177): Elementor\Core\Documents_Manager->register_types()
#5 ../wp-content/plugins/elementor/core/editor/editor.php(243): Elementor\Core\Documents_Manager->get(NULL)
#6 ../wp-content/plugins/elementor/includes/frontend.php(207): Elementor\Core\Editor\Editor->is_edit_mode()
#7 ../wp-includes/class-wp-hook.php(324): Elementor\Frontend->init_render_mode('')
#8 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#9 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#10 ../wp-includes/template-loader.php(13): do_action('...')
#11 ../wp-blog-header.php(19): require_once('...')
#12 ../index.php(17): require('...')
#13 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-12-17 10:39:11 [error X 1][../wp-content/plugins/elementor-pro/modules/global-widget/module.php::202] Uncaught Error: Class "ElementorPro\Modules\GlobalWidget\Documents\Widget" not found in ../wp-content/plugins/elementor-pro/modules/global-widget/module.php:202
Stack trace:
#0 ../wp-includes/class-wp-hook.php(324): ElementorPro\Modules\GlobalWidget\Module->register_documents(Object(Elementor\Core\Documents_Manager))
#1 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#2 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#3 ../wp-content/plugins/elementor/core/documents-manager.php(728): do_action('...', Object(Elementor\Core\Documents_Manager))
#4 ../wp-content/plugins/elementor/core/documents-manager.php(177): Elementor\Core\Documents_Manager->register_types()
#5 ../wp-content/plugins/elementor/core/editor/editor.php(243): Elementor\Core\Documents_Manager->get(NULL)
#6 ../wp-content/plugins/elementor/includes/frontend.php(207): Elementor\Core\Editor\Editor->is_edit_mode()
#7 ../wp-includes/class-wp-hook.php(324): Elementor\Frontend->init_render_mode('')
#8 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#9 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#10 ../wp-includes/template-loader.php(13): do_action('...')
#11 ../wp-blog-header.php(19): require_once('...')
#12 ../index.php(17): require('...')
#13 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-12-18 09:00:40 [error X 1][../wp-content/plugins/elementor/core/base/module.php::350] Uncaught Error: Class "ElementorPro\Modules\GlobalWidget\Widgets\Global_Widget" not found in ../wp-content/plugins/elementor/core/base/module.php:350
Stack trace:
#0 ../wp-includes/class-wp-hook.php(324): Elementor\Core\Base\Module->init_widgets(Object(Elementor\Widgets_Manager))
#1 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#2 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#3 ../wp-content/plugins/elementor/includes/managers/widgets.php(146): do_action('...', Object(Elementor\Widgets_Manager))
#4 ../wp-content/plugins/elementor/includes/managers/widgets.php(345): Elementor\Widgets_Manager->init_widgets()
#5 ../wp-content/plugins/elementor/includes/elements/column.php(1038): Elementor\Widgets_Manager->get_widget_types('...')
#6 ../wp-content/plugins/elementor/includes/base/element-base.php(1502): Elementor\Element_Column->_get_default_child_type(Array)
#7 ../wp-content/plugins/elementor/includes/base/element-base.php(369): Elementor\Element_Base->get_child_type(Array)
#8 ../wp-content/plugins/elementor/includes/base/element-base.php(1547): Elementor\Element_Base->add_child(Array)
#9 ../wp-content/plugins/elementor/includes/base/element-base.php(314): Elementor\Element_Base->init_children()
#10 ../wp-content/plugins/elementor/core/dynamic-tags/dynamic-css.php(39): Elementor\Element_Base->get_children()
#11 ../wp-content/plugins/elementor/core/dynamic-tags/dynamic-css.php(40): Elementor\Core\DynamicTags\Dynamic_CSS->render_styles(Object(Elementor\Element_Column))
#12 ../wp-content/plugins/elementor/core/files/css/post.php(178): Elementor\Core\DynamicTags\Dynamic_CSS->render_styles(Object(Elementor\Element_Section))
#13 ../wp-content/plugins/elementor/core/files/css/base.php(687): Elementor\Core\Files\CSS\Post->render_css()
#14 ../wp-content/plugins/elementor/core/files/base.php(180): Elementor\Core\Files\CSS\Base->parse_content()
#15 ../wp-content/plugins/elementor/core/files/css/base.php(133): Elementor\Core\Files\Base->update_file()
#16 ../wp-content/plugins/elementor/core/files/css/base.php(229): Elementor\Core\Files\CSS\Base->update()
#17 ../wp-content/plugins/elementor/core/files/css/post.php(201): Elementor\Core\Files\CSS\Base->enqueue()
#18 ../wp-content/plugins/elementor/core/dynamic-tags/manager.php(479): Elementor\Core\Files\CSS\Post->enqueue()
#19 ../wp-includes/class-wp-hook.php(324): Elementor\Core\DynamicTags\Manager->after_enqueue_post_css(Object(Elementor\Core\DynamicTags\Dynamic_CSS))
#20 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#21 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#22 ../wp-content/plugins/elementor/core/files/css/base.php(276): do_action('...', Object(Elementor\Core\Files\CSS\Post))
#23 ../wp-content/plugins/elementor/core/files/css/post.php(201): Elementor\Core\Files\CSS\Base->enqueue()
#24 ../wp-content/plugins/elementor-pro/modules/theme-builder/classes/locations-manager.php(148): Elementor\Core\Files\CSS\Post->enqueue()
#25 ../wp-includes/class-wp-hook.php(324): ElementorPro\Modules\ThemeBuilder\Classes\Locations_Manager->enqueue_styles('')
#26 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#27 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#28 ../wp-includes/script-loader.php(2263): do_action('...')
#29 ../wp-includes/class-wp-hook.php(324): wp_enqueue_scripts('')
#30 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#31 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#32 ../wp-includes/general-template.php(3065): do_action('...')
#33 ../wp-content/themes/hello-elementor/header.php(24): wp_head()
#34 ../wp-includes/template.php(810): require_once('...')
#35 ../wp-includes/template.php(745): load_template('...', true, Array)
#36 ../wp-includes/general-template.php(48): locate_template(Array, true, true, Array)
#37 ../wp-content/plugins/elementor/modules/page-templates/templates/header-footer.php(9): get_header()
#38 ../wp-includes/template-loader.php(106): include('...')
#39 ../wp-blog-header.php(19): require_once('...')
#40 ../index.php(17): require('...')
#41 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 2 of 2JS: 2024-11-21 19:55:22 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.25.9:3:956093] elementorFrontend is not defined 
JS: 2024-12-09 11:46:55 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.25.10:3:817660] elementor.getCurrentElement().model is undefined 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Die Kompatibilität ist nicht angegeben

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by tomandersen** [comment]
As a guess, try turning off "Element Caching" under Elementor -> Settings -> Features.

**Comment by nicholaszein** [comment]
Hello there @musikblog!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Core v3.26.2! 🥳 

**We reverted back the activation of Element Caching on existing websites.**

To learn how to make the most of Elementor with Element Caching Active and improve your website's performance without breaking dynamic content inserted via shortcodes or dynamic tags, please read the document below and learn how to set Element Caching as **Inactive** individually per widget.

https://elementor.com/help/element-caching-help/

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
